### ISSUE TITLE
Add titles to inputs

### ISSUE THREAD

**Issue by tsostarics** [tsostarics/PitchMendR#49]
Setting `title = "x"` to an input will add information when the user hovers over the element. These can be useful for quick descriptions of various inputs (see #48) and I believe are used with screen reading software, so for accessibility reasons they should be added to all inputs. When an element doesn't have a dedicated title parameter, you can wrap the whole input in a span and set the title there.
